main.js

```diff
@@ -0,0 +1,23 @@
+document.querySelector("li").remove();
+document.querySelector("span").remove();
+document.querySelector("main .posts article").remove();
```

Comment:
Pretty sure this'll remove the first article from the posts, not the last one. Gotta remove the last one.